### PR TITLE
fix rolebinding describer

### PR DESCRIPTION
**Pull request by deads2k**
The describer wasn't handling references to cluster roles properly.

@liggitt ptal

### CODE REVIEW

File: pkg/cmd/server/bootstrappolicy/policy.go [hunk 1]
```diff
@@ -104,6 +104,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			Rules: []authorizationapi.PolicyRule{
 				{Verbs: util.NewStringSet("get"), Resources: util.NewStringSet("users"), ResourceNames: util.NewStringSet("~")},
 				{Verbs: util.NewStringSet("list"), Resources: util.NewStringSet("projectrequests")},
+				{Verbs: util.NewStringSet("list", "get"), Resources: util.NewStringSet("clusterroles")},
```
Comment (liggitt) [inline pkg/cmd/server/bootstrappolicy/policy.go]: should `EditRoleName` and `ViewRoleName` include list/get on roles?

**Comment by liggitt** [comment]
question about list/get permissions on roles within a namespace, LGTM otherwise

File: pkg/cmd/server/bootstrappolicy/policy.go (hunk 1 above)
Comment (deads2k) [inline pkg/cmd/server/bootstrappolicy/policy.go]: Why would they?  I can see a need for that when you're trying to correctly map your users to roles (the admin doing the binding).

Comment (liggitt) [inline pkg/cmd/server/bootstrappolicy/policy.go]: yeah, nevermind

**Comment by deads2k** [comment]
[merge]

**Pull request merged**